internal/pkg/cli/run_local.go

```diff
@@ -223,6 +237,10 @@ func newRunLocalOpts(vars runLocalVars) (*runLocalOpts, error) {
 		}
 		return containerURIs, nil
 	}
+	o.debounceTime = 5 * time.Second
```

Comment:
why does this need to be a field of `runLocalOpts` 👀

Reply:
For tests to set the debounce time to 0